beacon/app/beacon.hs

```diff
@@ -514,5 +551,15 @@ parseOpts =   BenchmarksCompareOptions
     parseOverwriteData = flag OverwriteData DoNotOverwriteData
                        $ mconcat
                          [ long "overwrite-data"
-                         , help "Overwrite previous benchmarkd data"
+                         , help "Overwrite previous benchmark data"
                          ]
+
+--------------------------------------------------------------------------------
+-- Functions to deal with input
+--------------------------------------------------------------------------------
+
+-- | Sanitize an input string so that it can be used to specify a file path.
+--
+-- TODO: We only consider Linux at the moment..
+toFilePath :: String -> String
+toFilePath = replace "/" "-"
```

Comment:
The closest existing thing that comes to mind is a [slug](https://en.wikipedia.org/wiki/Clean_URL#Slug); eg from [`toSlug`](https://hackage.haskell.org/package/slugger-0.1.0.2/docs/Data-String-Slugger.html#v:toSlug) from `slugger`:
```haskell
 Λ import Data.String.Slugger
 Λ toSlug "a/b/c"
"a-b-c"
```
But `replace "/" "-"` might be totally fine for now.

Reply:
It's better to use a library I think. I'll add a dependency on `slugger`. Thank you for pointing this out!